platform/security/filter/security-filter-csrf/src/main/java/org/codice/ddf/security/filter/csrf/CsrfFilter.java

```diff
@@ -69,7 +69,10 @@ public void init(FilterConfig filterConfig) throws ServletException {
     String externalHttpPort = SystemBaseUrl.EXTERNAL.getHttpPort();
     String externalHttpsPort = SystemBaseUrl.EXTERNAL.getHttpsPort();
 
-    trustedAuthorities = new ArrayList<>();
+    if (trustedAuthorities == null) {
+      trustedAuthorities = new ArrayList<>();
```

Comment:
✏️ Nit, you could move the`trustedAuthorities` and the `protectedContexts` initialization into a default constructor.